nuxeo/templates/deployment.yaml

```diff
@@ -178,7 +178,7 @@ spec:
 {{- end }}
 {{- with .Values.packages }}
         - name: NUXEO_PACKAGES
-          value: {{ toYaml . }}
+          value: {{ . }}
```

Comment:
Strange we didn't see it before!

Reply:
It only happens when the line is very long.